src/settings/NotifTroubleshootingScreen.js

```diff
@@ -289,6 +294,49 @@ function jsonifyNotificationReport(report: NotificationReport): string {
   );
 }
 
+// See https://chat.zulip.org/#narrow/stream/107-kandra/topic/notification.20plan.20expiration.20warnings/near/1719677
+export const kPushNotificationsEnabledEndDoc: URL = new URL(
+  'https://zulip.com/help/self-hosted-billing#upgrades-for-legacy-customers',
```

Comment:
This link is to a private stream. Generally we don't put such a reference into our codebase.

In this case, I think no reference is needed. The information I take from following the link is basically that this is the link Alya asks to put here. That's the kind of information that can be relevant at PR discussion time, or perhaps in a commit message, but isn't needed in the tree itself. (And we can say it directly, rather than via a link.)

Reply:
Oops, indeed; thanks for the catch.